client/landing/gutenboarding/steps.ts

```diff
@@ -6,6 +6,7 @@ import { map } from 'lodash';
 export enum Step {
 	IntentGathering = '/',
 	DesignSelection = '/design',
+	CreateSite = '/design',
```

Comment:
I don't think we want two enums with the same value, do we? This would make the paths collide:

```suggestion
	CreateSite = '/create-site',
```

We'll need to handle that in the routing, staying on the same step should be fine for now.